lldb/source/Commands/Options.td

```diff
@@ -675,6 +675,8 @@ let Command = "platform shell" in {
 let Command = "process launch" in {
   def process_launch_stop_at_entry : Option<"stop-at-entry", "s">,
     Desc<"Stop at the entry point of the program when launching a process.">;
+  def process_launch_stop_at_main : Option<"stop-at-user-entry", "m">,
+    Desc<"Stop at the user entry point when launching a process.">;
```

Comment:
It would be fine to mention "main" here something like:
```
Desc<"Stop at the user entry point when launching a process. For C based languages this will be the 'main' function, but this might differ for other languages.">;
```